---
layout: post
title:  Working in the Genome Browser
description: Viewing and mining mulit-omic data in the context of a reference genome
date:   2023-05-17 15:01:35 +0300
image:  '/images/gbrowse2.jpg'
tags:   [web-based, point-and-click, dataviz]
---

## Registration and location

The workshop will take place in the [Holman Reading Room](https://www.library.upenn.edu/holman) in the Biomedical library, and is free to UPenn graduate students, postdocs, faculty, and staff.

The workshop will take place over two days (Wed, May 17 and Thur, May 18th).  Registration is required for both days, and seating is limited.

## Overview

[The UCSC Genome Browser](https://genome.ucsc.edu) is an omics data consolidator, graphical viewer, and general bioinformatics resource for the genomics community.  With the ever-increasing amounts of data being generated each year, tools like the UCSC Genome Browser are increasingly playing a key step in data analysis and interpretation. The Genome Browser resource services over 1.4 million users per year, and offers over 6000 tracks on the two latest human GRCh assemblies alone, GRCh38/hg38 and GRCh37/hg19. There are also over 200 assemblies available on the Genome Browser. The resource supports over 30 data formats such as bed/bigBed, wig/bigWig, VCF and GTF/GFF, thus allowing users to not only display their own annotations, but also to visualize data from a large number of sources in a single location.

This workshop will be split into two consecutive days, with the first day focusing on key features of the Genome Browser, and the second day devoted to use of this resource for mining clinical data.

## About your instructor

Robert Kuhn received his PhD in biochemistry and molecular biology at the University of California Santa Barbara, where he studied the centromeres of yeast.  Following a postdoctoral fellowship studying photoreceptor genes in plants at UC Berkeley, he taught biochemistry, genetics and molecular biology at the University of California Santa Cruz.

He joined the UCSC Genome Browser project in 2003, retiring as Associate Director in 2022.  The Genome Browser is a widely used visualization tool giving access to the genomes of human, model organisms and more than one hundred other animals.  Dr. Kuhn's responsibilities included influencing the growth and interface of the Browser, identifying important datasets for inclusion into the Browser and enabling researchers worldwide through in-person workshops and seminars as well as the YouTube channel.  He has now taught more 300 presentations (https://bit.ly/kuhnTalks).  He has a particular interest in the support of clinical geneticists via online databases and tools for interpreting data.

Dr Kuhn is now operating as Robert Kuhn Consulting, where he continues to offer Browser trainings, bioinformatics consultation and educational materials.


## Agenda

<details>
  <summary>Day 1</summary>


**9:00 - 10:30: Intro to the UCSC Genome Browser – features and navigation.**

A tour of the basic parts of the UCSC Genome Browser
* General overview of the Browser, what it is, history
* Variants – benign and pathogenic, large and small (CNV, SNV)
* Display configuration
* Track settings
* Exporting Browser .pdf images for publication
* Sample searches: Gene of interest, regions of interest
* HGVS – jumping directly to genes, codons
* Navigation:  drag-and-zoom, drag-reorder tracks, switching genome assemblies and organisms via homology, highlighting
* Keeping track of your thinking by saving and sharing sessions
* Resources for education
* Using the Browser in teaching genetics and molecular biology
* YouTube channel

**10-30 - 10:45: Coffee break**

**10:45 - 12:00: Genome Browser data**

* Publications / gene interactions
* Data – genesets, annotation, 1000 Genomes, CRISPR guides, pathways, epigenomics
* Variants – CNVs and SNPs, benign and pathogenic
* Understanding various SNP tracks – composite tracks
* Finding more information about a track or track item
* Table Browser and extracting data
* Getting gene sequences
* Finding the table that goes with your track
* Subsets / Filters
* Intersections
* Custom tracks - uploading and displaying your own data in the Browser
* Data types/formats – BED files (basic annotation), wiggle (RNA-seq)
* Configuration
* How to get DNA for the region you are viewing
* BLAT and isPCR (using the DNA)
* Comparative genomics – multispecies alignments
* SARS-CoV2 browser

**12:00 - 1:00: Lunch**

**1:00 - 2:00: Still more features**

* More data types – barChart, interact, Hi-C, lollipop
* Custom tracks and track hubs – uploading and displaying your own data
* Sequencing data:  coverage (BAM files), variant calls (VCF), RNA-seq (wiggle)
* Public track hubs
* Track collections – co-configuring multiple wiggle tracks

2:00 - 4:00: Cementing your knowledge:  Working through a problem set + Q&A

* Problem set for hands-on reinforcement of how to use the Browser
* Participant-directed review of features and data (Q&A)
* One-on-one consultation/troubleshooting
* Examples of comparative genomics, changes between assemblies, evolution
</details>

**DAY 2**

**9:00 - 10:30: Getting more out of the Browser + Clinical context**

* Naming of genomes – hg19 vs hg38 + T2T
* Advantages and drawbacks of using the various assemblies
* Recommended Track Sets – quick display of most valuable datasets for interpreting variants large and small (CNVs, SNVs)
* DECIPHER, DGV
* GnomAD, OMIM, GTEx expression data, UniProt
* Table Browser – extracting coordinates from gene names
* Multi-Region mode
* Exon-only display
* Viewing discontinuous regions (e.g., pathways)
* Alternate haplotypes in human genome assemblies
* Data Integrator – combining data from multiple tables

**10-30 - 10:45: Coffee break**

**10:45 - 12:00: Deeper dive into Recommended Track Sets**

* CNVs and SNVs – benign and pathogenic
* Individual tracks and their usefulness in variant interpretation
* What has been seen before and what was said about it
* What data can be used to think about a novel variant
* Table Browser example – combining trios
* Custom Tracks – pgSNP and VCF formats
* Variant Annotation Integrator
* Predicting the effects of variants

**12:00 - 1:00: Lunch**

**1:00 - 4:00: Cementing your knowledge - Advanced problem session to work on example problems**

* Problem set for hands-on reinforcement of how to use the Browser
* Participant-directed review of features and data (Q&A)
* Concurrent:  One-on-one consultation/troubleshooting

## Workshop resources

* [2023 NAR paper](https://academic.oup.com/nar/article/51/D1/D1188/6845436?login=false) - this is the most current publication describing the UCSC genome browser features.
* [Recommended tracks paper](https://onlinelibrary.wiley.com/doi/10.1002/humu.24335) - this paper describes a set of recommended tracks for interpreting clincial variant data
* [Small file for loading SNV custom track](https://users.soe.ucsc.edu/~kuhn/workshops/philly2023/hg38.snvs.txt)
* [Small file with several custom tracks in various formats](https://users.soe.ucsc.edu/~kuhn/workshops/philly2023/ctExamples.txt)
* [Simple example gene list](https://users.soe.ucsc.edu/~kuhn/workshops/philly2023/genelist)
* [File for bar chart](https://users.soe.ucsc.edu/~kuhn/workshops/philly2023/barChart.txt)
* [Hands-on exercise](https://gemini-series.netlify.app/code/practicalPhilly2023.pdf) - this is the series of exercises that we began working through in the afternoon on day 1 of the workshop.  This exercise uses this [example bam track file](https://users.soe.ucsc.edu/~kuhn/workshops/bamDir/bamTracks.txt).


## Recordings

[Day 1 - morning](https://pennmedicine.zoom.us/rec/share/aPdoYinbK1moLJI9bMpd-L0lcCU2MNZUis9MXdrprlFsPKA3xsmsA9591kQ8vuPz.SR_5pHWrnIYPvUa5?startTime=1684327497000)
Passcode: R9GN%d83

[Day 1 - afternoon](https://pennmedicine.zoom.us/rec/share/aPdoYinbK1moLJI9bMpd-L0lcCU2MNZUis9MXdrprlFsPKA3xsmsA9591kQ8vuPz.SR_5pHWrnIYPvUa5?startTime=1684347238000)
Passcode: R9GN%d83



